Core/HTTPSUpgrade.swift

```diff
@@ -56,66 +49,29 @@ public class HTTPSUpgrade {
             completion(false)
             return
         }
         
-        let isLocallyUpgradable = !isLocalListReloading() && isInLocalUpgradeList(host: host)
-        os_log("%s %s locally upgradable", log: generalLog, type: .debug, host, isLocallyUpgradable ? "is" : "is not")
-        if  isLocallyUpgradable {
-            Pixel.fire(pixel: .httpsLocalUpgrade)
-            completion(true)
-            return
-        }
-        
-        isInServiceUpgradeList(host: host) { result in
-            os_log("%s %s service upgradable", log: generalLog, type: .debug, host, result.isInList ? "is" : "is not")
-            if result.isInList {
-                Pixel.fire(pixel: result.isCached ? .httpsServiceCacheUpdgrade : .httpsServiceRequestUpgrade)
-                completion(true)
-            } else {
-                Pixel.fire(pixel: result.isCached ? .httpsServiceCacheNoUpdgrade : .httpsServiceRequestNoUpdgrade)
-                completion(false)
-            }
-        }
+        waitForAnyReloadsToComplete()
+        let isUpgradable = isInUpgradeList(host: host)
+        os_log("%s %s upgradable", log: generalLog, type: .debug, host, isUpgradable ? "is" : "is not")
+        Pixel.fire(pixel: isUpgradable ? .httpsLocalUpgrade : .httpsNoUpgrade)
+        completion(isUpgradable)
+           
     }
     
-    private func isInLocalUpgradeList(host: String) -> Bool {
+    private func isInUpgradeList(host: String) -> Bool {
         guard let bloomFilter = bloomFilter else { return false }
         return bloomFilter.contains(host)
     }
     
-    private func isInServiceUpgradeList(host: String, completion: @escaping ServiceCompletion) {
-        let sha1Host = host.sha1
-        let partialSha1Host = String(sha1Host.prefix(4))
-        let url = appUrls.httpsLookupServiceUrl(forPartialHost: partialSha1Host)
-        let serviceRequest = APIRequest.urlRequestFor(url: url, timeoutInterval: 10.0)
-        let cachedResponse = URLCache.shared.cachedResponse(for: serviceRequest)?.response as? HTTPURLResponse
-        
-        APIRequest.request(url: url,
-                           timeoutInterval: 10,
-                           callBackOnMainThread: true) { (response, _) in
-            
-            guard let httpResponse = response?.urlResponse as? HTTPURLResponse, let data = response?.data else {
-                completion((isInList: false, isCached: false))
-                return
-            }
-            
-            let result = try? JSONDecoder().decode([String].self, from: data)
-            let isCached = isResponseFromCache(httpResponse, cachedResponse: cachedResponse)
-            let isInList = result?.contains(sha1Host) ?? false
-            completion((isInList: isInList, isCached: isCached))
-        }
-    }
-    
-    private func isLocalListReloading() -> Bool {
-        if !dataReloadLock.try() {
-            return true
-        }
+    private func waitForAnyReloadsToComplete() {
```

Comment:
Adding wait that was previously in the app now that we can no longer fallback to backend service